source/MaterialXTest/MaterialXGenShader/GenShaderUtil.cpp

```diff
@@ -788,6 +792,7 @@ void ShaderGeneratorTester::validate(const mx::GenOptions& generateOptions, cons
 
                 mx::string elementName = mx::replaceSubstrings(namePath, pathMap);
                 elementName = mx::createValidName(elementName);
+                elementName = mx::replaceSubstrings(elementName, filenameRemap);
```

Comment:
Need to do this to remove namespace separators which are valid in an Element name but bad as a file name.